lib/addDefaults.js

```diff
@@ -428,6 +429,47 @@ function optimizeForCesium(gltf) {
     }
 }
 
+function inferBufferViewTargets(gltf) {
+    // If bufferView elements are missing targets, we can infer their type from their use
+    var needsTarget = {};
+    var shouldTraverse = false;
```

Comment:
Is this optimization worth it: change this boolean to an integer with the number of bufferViews that need a `target`, decrease it and set `needsTarget[bufferViewId] = false` every time a target is set, then exit the outer most `ForEach.mesh` early when it is zero.  The return value of the callback could indicate `break` if it doesn't already.

Given that this is simple, it could be worth it, I just think most models will either always have target defined or never have target defined.